indexing-service/src/test/java/io/druid/indexing/overlord/RemoteTaskRunnerTest.java

```diff
@@ -127,8 +127,9 @@ public void testRun() throws Exception
     mockWorkerCompleteSuccessfulTask(task);
     Assert.assertTrue(workerCompletedTask(result));
 
-    Assert.assertEquals(task.getId(), result.get().getId());
-    Assert.assertEquals(TaskStatus.Status.SUCCESS, result.get().getStatusCode());
+    TaskStatus status = result.get(TIMEOUT_SECONDS, TimeUnit.SECONDS);
```

Comment:
I don't understand how this fixes something?

Adding a timeout to the `get` should only make possibly more test exceptions happen.